### PR TITLE
Create an RSS Feed

### PR DESCRIPTION
**Pull request by drapergeek**
Yes, I'm old, I use RSS.

Feel free to add a link for those who don't go looking for `feed.xml`.

Please and thank you!

### CODE REVIEW

File: config.rb [hunk 1]
```diff
@@ -40,3 +40,5 @@
 activate :livereload
 activate :neat
 activate :syntax
+
+page '/feed.xml', layout: false
```
Comment (drapergeek) [inline config.rb]: I followed the existing style. Happy to update if the maintainer prefers though.

Comment (adarsh) [inline config.rb]: Agreed. I turned Hound on but forgot to actually add any config.

**Comment by adarsh** [comment]
Merged this as d85da34 and feed available at http://adarsh.io/feed.xml.

What do you use as a reader these days?

**Comment by drapergeek** [comment]
http://feedly.com/ as the backend, http://reederapp.com/ios/ as the actual reading. Haven't had any complaints at all for either.

**Pull request closed without merging**